scripts/setup_runners.sh

```diff
@@ -0,0 +1,62 @@
+# MIT License
+#
+# Copyright (c) 2020 Shyam Jesalpura and EASE lab
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+# uninstall old versions
+sudo apt-get remove docker docker-engine docker.io containerd runc
+
+# download and install docker
+sudo apt-get update
+
+sudo apt-get install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+sudo add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+
+sudo apt-get update
+sudo apt-get install --yes docker-ce docker-ce-cli containerd.io 
+
+sudo groupadd docker
+sudo usermod -aG docker $USER
+newgrp docker 
+
+# Build docker image
+DOCKER_BUILDKIT=1 docker build \
+--progress=plain \
+--file ./Dockerfile.integration_test \
+--tag vhive_integration_test:latest .
+
+# start container
+docker run --rm -it --privileged \
```

Comment:
I think that you should separate the script that sets up the host from the script that starts the runners. for example, to reclaim storage, we would need to reboot the machine every day and restart the runners.

Reply:
done